frontend/appflowy_flutter/lib/workspace/application/settings/appearance/desktop_appearance.dart

```diff
@@ -48,7 +49,9 @@ class DesktopAppearance extends BaseAppearance {
 
     // Due to Desktop version has multiple themes, it relies on the current theme to build the ThemeData
     return ThemeData(
-      visualDensity: VisualDensity.standard,
+      visualDensity: UniversalPlatform.isMacOS || UniversalPlatform.isMobile
```

Comment:
Would it affect the margin or padding of the buttons?

Reply:
Reverted. Windows system font is correctly aligned so we're gonna leave it as it is right now